<launch>
    <arg name="use_real_ships" default="false" />
    <param name="use_real_ships" type="bool" value="$(arg use_real_ships)" />
    <node pkg="local_pathfinding" type="MOCK_wind_sensor.py" name="MOCK_wind_sensor" />
    <node pkg="local_pathfinding" type="MOCK_controller_and_boat.py" name="MOCK_controller_and_boat" />
    <node pkg="local_pathfinding" type="MOCK_AIS.py" name="MOCK_AIS" />
    <node pkg="local_pathfinding" type="MOCK_global_planner.py" name="MOCK_global_planner" />
    <node pkg="local_pathfinding" type="MOCK_UDP_bridge.py" name="MOCK_UDP_bridge" />
</launch>
